### ISSUE TITLE
Fail to import an existing/new Project

### ISSUE THREAD

**Issue by sfjohansson** [consulo/consulo#208]
![screen shot 2016-08-09 at 09 42 59](https://cloud.githubusercontent.com/assets/912814/17510229/f629675e-5e15-11e6-9d29-12e4e77b5edb.png)

Build 1384

This seem to apply to both PC and Mac

**Comment by VISTALL** [comment]
Can you list files in this directory?

**Comment by sfjohansson** [comment]
It is just a new empty project with an asset from the asset store imported...tried with different projects..

**Comment by VISTALL** [comment]
Can you attach it?  I will test it after work

**Comment by sfjohansson** [comment]
Just create a new unity project...nothing exciting in it =O

**Comment by VISTALL** [comment]
:) please attach it. I can make something wrong :)

**Comment by sfjohansson** [comment]
Certainly...here you go! :)

[EnumGenerator.zip](https://github.com/consulo/consulo/files/408616/EnumGenerator.zip)

**Comment by VISTALL** [comment]
Thanks

**Comment by VISTALL** [comment]
You have installed&enabled unity3d plugin? i don't have any problems with import

Thanks

**Comment by sfjohansson** [comment]
I did have in one of the projects...same result...let me try again.

**Comment by sfjohansson** [comment]
I rolled back to 1361...and there it works perfectly ...so when u say unity plugin...do you mean the one in the unity project or the one inside "consulo"?

The plugin inside consulo should have been enabled in all cases ( at least in theory unless it was not functioning correctly on my machine on the latest build by some reason).

**Comment by VISTALL** [comment]
Inside Consulo.

I will check. Sorry for issue (i thk it's my breaking change)

Better autoupdate will come in 2.0 version (maybe in autumn)

**Comment by sfjohansson** [comment]
Just tried a new update on my rollback..and it seems to be working better...although I do get this pop up:

![screen shot 2016-08-09 at 18 02 58](https://cloud.githubusercontent.com/assets/912814/17525690/b442b4f8-5e5b-11e6-91e1-e0c9ab4eabcb.png)

![screen shot 2016-08-09 at 18 06 19](https://cloud.githubusercontent.com/assets/912814/17525744/05092656-5e5c-11e6-8c6a-f744bc02df17.png)

Does that refer to the plug in for consulo or for Unity?

**Comment by VISTALL** [comment]
https://github.com/consulo/UnityEditorConsuloPlugin

**Comment by VISTALL** [comment]
It client library for UnityEditor (running tests, sending console output to consulo, etc)

**Comment by sfjohansson** [comment]
I do have that... =O

**Comment by VISTALL** [comment]
I think you have old version

**Comment by VISTALL** [comment]
Oh. It search on default path 'Assets/Editor/Plugins'

**Comment by sfjohansson** [comment]
trying to move it here...to see if makes the difference..also grabbed latest

**Comment by sfjohansson** [comment]
moved into that folder..and now it says outdated...although I did grab it just from the repo...I'm on mac

**Comment by VISTALL** [comment]
You can click Update - and will be ok

**Comment by sfjohansson** [comment]
seem to have solved it :)

**Comment by sfjohansson** [comment]
Cheers! =)

**Comment by VISTALL** [comment]
Ok. Thanks
